terraform/modules/vpc/main.tf

```diff
@@ -0,0 +1,46 @@
+resource "aws_vpc" "main" {
```

Comment:
Consider enabling VPC Flow Logs for the VPC to ensure network monitoring and security compliance with SOC2. This can be done by adding an `aws_flow_log` resource.